docs/builders/email_landing_page.rst

```diff
@@ -127,6 +127,35 @@ If you search through the list of available Themes, the new MJML Themes ``Brienz
 
 To learn more about creating Themes please :doc:`check the documentation</builders/creating_themes>`. 
 
+Custom fonts
+************
+
+The Style Manager > Typography > Fonts list can be extended to include custom fonts.
```

Comment:
```suggestion
From Mautic 4.4.9 you can extend the Style Manager > Typography > Fonts list to include custom fonts.
```

Vale fix for grammar, and also mentioning the version from which you will be able to do this (assuming it goes into 4.4.9 and also 5.x).